<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrapbookGUI</class>
 <widget class="QWidget" name="ScrapbookGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>410</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="scrapbookTab">
      <attribute name="title">
       <string>Scrapbook</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0" colspan="6">
        <widget class="QListWidget" name="scrapbookArea"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="viewButton">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Export to HTML</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="blogTab">
      <attribute name="title">
       <string>Blog</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="blogStackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_5"/>
         <widget class="QWidget" name="page_6"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tweetTab">
      <attribute name="title">
       <string>Tweet</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="tweetStackedWidget">
         <widget class="QWidget" name="page_3"/>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="multimediaTab">
      <attribute name="title">
       <string>Multimedia</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="multimediaStackedWidget">
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
